crates/turbopack-ecmascript/src/parse.rs

```diff
@@ -138,6 +141,19 @@ pub async fn parse(
     let fs_path = &*source.ident().path().await?;
     let file_path_hash = *hash_ident(source.ident().to_string()).await? as u128;
     let ty = ty.into_value();
+    let tsconfig = find_context_file(source.ident().path(), tsconfig());
+    let tsconfig = match *tsconfig.await? {
+        FindContextFileResult::Found(path, _) => Some(
+            read_tsconfigs(
+                path.read(),
+                SourceAssetVc::new(path).into(),
+                node_cjs_resolve_options(path.root()),
+            )
+            .await?,
+        ),
+        FindContextFileResult::NotFound(_) => None,
+    };
```

Comment:
This causes a lot of errors in front as tsconfig in node_modules is read:

```
  C:\Repos\front\apps\vercel-site\node_modules\.pnpm\@vercel+analytics@0.1.9-beta.3_react@18.2.0\node_modules\@vercel\analytics\tsconfig.json
    An issue occurred while parsing a tsconfig.json file.
    extends doesn't resolve correctly
```

A tsconfig shouldn't be read in `parse`